src/pyuvdata/telescopes.py

```diff
@@ -400,30 +402,79 @@ def __init__(self):
-            "(indicating east/west orientation) and 'north (indicating "
-            "north/south orientation)."
+            "Antenna diameters in meters. Used by CASA to "
+            "construct a default beam if no beam is supplied."
         )
-        self._x_orientation = uvp.UVParameter(
-            "x_orientation",
+        self._antenna_diameters = uvp.UVParameter(
+            "antenna_diameters",
             description=desc,
             required=False,
+            form=("Nants",),
+            expected_type=float,
+            tols=1e-3,  # 1 mm
+        )
+
+        desc = (
+            'Antenna mount type, shape (Nants,). Options are: "alt-az", "equitorial", '
+            '"orbiting", "x-y", "alt-az+nasmyth-r", "alt-az+nasmyth-l", "phased", '
+            '"fixed", and "other".'
+        )
+        self._mount_type = uvp.UVParameter(
+            "mount_type",
+            description=desc,
+            form=("Nants",),
+            required=False,
             expected_type=str,
-            acceptable_vals=["east", "north"],
+            acceptable_vals=[
+                "alt-az",
+                "equatorial",
+                "orbiting",
+                "x-y",
+                "alt-az+nasmyth-r",
+                "alt-az+nasmyth-l",
+                "phased",
+                "fixed",
+                "other",
+            ],
+        )
+
+        self._Nfeeds = uvp.UVParameter(
+            "Nfeeds",
+            description="Number of feeds.",
+            expected_type=int,
+            acceptable_vals=[1, 2],
+            required=False,
         )
 
         desc = (
-            "Antenna diameters in meters. Used by CASA to "
-            "construct a default beam if no beam is supplied."
+            "Array of feed orientations. shape (Nants, Nfeeds), type str. Options are: "
+            "x/y or r/l. Optional parameter, required if feed_angle is set."
         )
-        self._antenna_diameters = uvp.UVParameter(
-            "antenna_diameters",
+        self._feed_array = uvp.UVParameter(
+            "feed_array",
             description=desc,
             required=False,
-            form=("Nants",),
+            expected_type=str,
+            form=("Nants", "Nfeeds"),
+            acceptable_vals=["x", "y", "r", "l"],
+        )
+
+        desc = (
+            "Position angle of a given feed, shape (Nants, Nfeeds), units of radians. "
+            "The feed angle is defined with respect to the great circle going through "
+            "constant azimuth and zenith (for pointed telescopes) or north (for fixed "
+            "telescopes like HERA or MWA), where a feed angle of 0 would correspond to "
+            "vertical polarization for linear feeds, and pi/2 to horizontal "
+            "polarization (as seen by the observer with reference to a horizontal "
+            "coordinate system). Optional parameter, required if feed_array is set."
```

Comment:
```
            "where a feed angle of 0 would correspond to "
            "vertical polarization for linear feeds, and pi/2 to horizontal "
            "polarization (as seen by the observer with reference to a horizontal "
            "coordinate system)
```

I find this very confusing. I don't consider either MWA dipole to have a vertical polarization. I wasn't sure what you meant by a horizontal coordinate system, Wikipedia say that's equivalent to alt/az coordinates. In that frame, both North and East aligned dipoles would be horizontal, not vertical...

Reply:
I can try another pass here, but I think I would call the polarization aligned with north-south plane as being "vertical". This gets weird though when talking about zenith (as all good spherical geometry problems do), so maybe this calls for a more extended discussion _also_ in the "conventions"  page.

Reply:
North-South could be vertical in some frames but not in the horizontal reference plane. I'd like to just stay away from the word vertical and go with something more like 0 means North for fixed mounts and aligned with the great circle for pointed telescopes, 90 means East for fixed mounts and perpendicular to the great circle for pointed telescopes.

Reply:
Okay, I've updated it in 11d373ecdf270b073d714514c43f9a1bd55e21d0.